ftw/mail/configure.zcml

```diff
@@ -42,6 +42,12 @@
         permission="zope2.View"
         />
 
+    <!-- Mail object creator -->
+    <adapter
+        for="*
+             zope.publisher.interfaces.browser.IBrowserRequest"
```

Comment:
@deiferni ahhh nevermind: `MailInbound` is a BrowserView, so you got a request with applied layers 👍